transfers/transfer.py

```diff
@@ -41,41 +44,58 @@ def fsdecode(filename):
 
 THIS_DIR = os.path.abspath(os.path.dirname(__file__))
 sys.path.append(THIS_DIR)
-from models import Unit, Session
 
 LOGGER = logging.getLogger('transfer')
-# Configure logging
-CONFIG = {
-    'version': 1,
-    'disable_existing_loggers': False,
-    'formatters': {
-        'default': {
-            'format': '%(levelname)-8s  %(asctime)s  %(filename)s:%(lineno)-4s %(message)s',
-            'datefmt': '%Y-%m-%d %H:%M:%S',
-        },
-    },
-    'handlers': {
-        'console': {
-            'class': 'logging.StreamHandler',
-            'formatter': 'default',
+
+CONFIG_FILE = None
+
+
+def get_setting(setting, default=None):
+    config = configparser.SafeConfigParser()
+    try:
+        config.read(CONFIG_FILE)
+        return config.get('transfers', setting)
+    except:
+        return default
+
+
+def setup(config_file):
+    global CONFIG_FILE
+    CONFIG_FILE = config_file
+    models.init(get_setting('databasefile', os.path.join(os.path.dirname(__file__), 'transfers.db')))
+
+    # Configure logging
+    default_logfile = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'automate-transfer.log')
```

Comment:
This and default database file could use `THIS_DIR` instead of `os.path.abspath(os.path.dirname(__file__))`